packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -14,6 +14,7 @@ import {
     TelemetryEventPropertyType,
 } from "@fluidframework/common-definitions";
 import { BaseTelemetryNullLogger, performance } from "@fluidframework/common-utils";
+import { ContainerErrorType, IErrorBase } from "@fluidframework/container-definitions";
```

Comment:
IHMO, this dependency should not be added. Telemetry-utils should be generic layer.
I should be able to take dependency on it in any layer - drivers, container, runtime, without hitting layer checks.
Anything specific to container or runtime should ideally be in its own layer.

Reply:
Similar to changes to packages/loader/driver-utils/src/compliantLogging.ts, I'd keep DataCorruptionError in runtime-utils or container-utils (whatever is right) or would create a new package.

Reply:
commented below. i think this change is generally fine, just get rid of the reference